smartsim/_core/utils/helpers.py

```diff
@@ -515,3 +528,41 @@ def push_unique(self, fn: _TSignalHandlerFn) -> bool:
         if did_push := fn not in self:
             self.push(fn)
         return did_push
+
+    def _create_pinning_string(
+        pin_ids: t.Optional[t.Iterable[t.Union[int, t.Iterable[int]]]], cpus: int
+    ) -> t.Optional[str]:
+        """Create a comma-separated string of CPU ids. By default, ``None``
```

Comment:
Can we add params, returns, and raises doc strings to this?

Reply:
added